python/README.md

```diff
@@ -1,7 +1,21 @@
 # Svix
-The Svix client API documentation
 
-Please refer to [the documentation](https://docs.svix.com/) for usage instructions.
+The Svix client API
+
+## Usage
+
+```sh
+pip install Svix
+```
+
+```python
+from svix import Svix, ApplicationIn
```

Comment:
```suggestion
from svix.api import Svix, ApplicationIn
```

Reply:
This is where I started, but I changed it to this following the code in docs.svix.com
Which one is it? Should we update the docs then?

Reply:
We should update the docs then to suggest this, we haven't been exporting new types from `__init__`